recipes/backports.shutil_which/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "backports.shutil_which" %}
+{% set version = "3.5.1" %}
+{% set sha256 = "dd439a7b02433e47968c25a45a76704201c4ef2167deb49830281c379b1a4a9b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: pip install --no-deps .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - backports
+    - backports.shutil_which
+
+about:
+  home: https://github.com/minrk/backports.shutil_which
+  license: MIT
```

Comment:
Based on this [note]( https://github.com/minrk/backports.shutil_which/commit/496539f113d3ce2bdf9d737d4f16e176f73c2655 ) it should be MIT, PSF 2.